app/build.gradle

```diff
@@ -44,7 +44,8 @@ dependencies {
     compile 'com.android.support:design:23.1.0'
     compile 'com.squareup.okhttp:okhttp:2.5.0'
     compile 'com.jakewharton:butterknife:7.0.1'
-    
+    compile 'org.ligi:AXT:0.35'
```

Comment:
I know this is your baby and from quickly looking at it it's got a lot of nice stuff. But IMHO and more as a general point I think we should carefully consider if saving a few lines of code (13 in this case) ist worth adding another dependency..